activity_browser/app/ui/tabs/LCA_results_tabs.py

```diff
@@ -1006,12 +1015,22 @@ def build_export(self, has_table: bool = True, has_plot: bool = True) -> QWidget
 
     def calculate_MC_LCA(self):
         iterations = int(self.iterations.text())
+        if self.seed.text():
+            print('SEED: ', self.seed.text())
+            try:
+                seed = int(self.seed.text())
+            except ValueError:
+                traceback.print_exc()
+                QMessageBox.warning(self, 'Warning', 'Seed value must be an integer number or left empty.')
+                self.seed.setText('')
```

Comment:
This try/except can be avoided by using `self.seed.setValidator(QtGui.QIntValidator())` on line 950, which blocks anything except numbers from being entered into the field.